frame/system/src/lib.rs

```diff
@@ -361,7 +361,9 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn integrity_test() {
-			T::BlockWeights::get().validate().expect("The weights are invalid.");
+			sp_io::TestExternalities::default().execute_with(|| {
```

Comment:
Looks like it tries to compile this as no-std when benchmarks are enabled.  
This is only a test and should not end up in the runtime, right?  
So `#[cfg(feature = "std")]` should be fine?

Reply:
https://github.com/paritytech/substrate/pull/13094